constants/links.ts

```diff
@@ -43,7 +43,7 @@ export const EXTERNAL_LINKS = {
 	},
 	Explorer: {
 		Optimism: 'https://optimistic.etherscan.io/tx',
-		OptimismKovan: 'https://kovan-optimistic.etherscan.io/tx',
```

Comment:
Need to make sure we update this link in the table as well: https://github.com/Kwenta/kwenta/blob/c5e1e69b0bd41c22e6a3aed04ec9981155197020/sections/futures/TradingHistory/TradesHistoryTable.tsx#L117

Reply:
Thanks for the reminder. I made the change here. Can you see it?
https://github.com/Kwenta/kwenta/pull/1359/files#diff-67c44e4805a21ab8854d6af6609a9beeff1366b4668c1c8e86ce686910c112f7